python/semantic_kernel/connectors/ai/open_ai/services/utils.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) Microsoft. All rights reserved.
+import logging
+from typing import TYPE_CHECKING, Any
+
+from semantic_kernel.functions.kernel_function_metadata import KernelFunctionMetadata
+
+if TYPE_CHECKING:
+    from semantic_kernel.connectors.ai.function_call_behavior import FunctionCallConfiguration
+    from semantic_kernel.connectors.ai.open_ai.prompt_execution_settings.open_ai_prompt_execution_settings import (
+        OpenAIChatPromptExecutionSettings,
+    )
+
+logger = logging.getLogger(__name__)
+
+
+TYPE_MAPPER = {
+    "str": "string",
+    "int": "number",
+    "float": "number",
+    "bool": "boolean",
+    "list": "array",
+    "dict": "object",
+}
+
+
+def update_settings_from_function_call_configuration(
+    function_call_configuration: "FunctionCallConfiguration", settings: "OpenAIChatPromptExecutionSettings"
+) -> None:
+    """Update the settings from a FunctionCallConfiguration."""
+    if function_call_configuration.required_functions:
+        if len(function_call_configuration.required_functions) > 1:
+            logger.warning("Multiple required functions are not supported. Using the first required function.")
```

Comment:
What is the scenario where this would happen? Is it if there are multiple functions that have the same name as `function_fully_qualified_name`.

Reply:
This looks like it's a warning if the LLM returns back multiple functions per function call/tool call. Right now, only expect there to be one function call (ID) mapping to one function call. In the event it ever returned back more than one, we'd get this warning.